apps/labs/posts/welcome-tania-allard-as-labs-codirector.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Welcoming Tania Allard as Quansight Labs co-director'
+published: January 04, 2021
+author: ralf-gommers
+description: "Today I'm incredibly excited to welcome Tania Allard to Quansight as Co-Director of Quansight Labs. Tania is a well-known and prolific PyData community member. In the past few years she has been involved as a conference organizer (JupyterCon, SciPy, PyJamas, PyCon UK, PyCon LatAm, JuliaCon and more), as a community builder (PyLadies, NumFOCUS, RForwards), as a contributor to Matplotlib and Jupyter, and as a regular speaker and mentor. She also brings relevant experience in both industry and academia - she joins us from Microsoft where she was a senior developer advocate, and has a PhD in computational modelling."
+category: [Community]
+featuredImage:
+  src: /posts/welcome-tania-allard-as-labs-codirector/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/welcome-tania-allard-as-labs-codirector/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+---
+
+![Tania Allard.](/posts/welcome-tania-allard-as-labs-codirector/tania_allard.jpg)
```

Comment:
In the original blog website, this image of Tania was the header image. Currently, I have the image in the top of the blog body.

This image's resolution is too small to be the feature or hero image. Should I get a higher resolution version of the same image to make it the hero and feature image?

Reply:
I found [this image](https://drive.google.com/file/d/1anr6tJ4UiyZO-p8oYZ5gDrnXOB64E0m6/view) on gdrive. @trallard should we use this, or do you have a larger version of the original?

Reply:
@trallard Pinging just surface this thread for you. :)